src/plugins/node-builtins-plugin.js

```diff
@@ -1,31 +1,34 @@
-const BUILTINS = {
-	fs: `export default {}`,
-	http: `export default {}`,
-	https: `export default {}`,
-	url: `export default {}`,
-	util: `export default {}`,
-	zlib: `export default {}`
-};
+import { builtinModules } from 'module';
 
 /**
- * Stub out Node built-ins.
+ * Return an error on Node built-ins.
```

Comment:
Nit: It doesn't always return an error after your latest changes.

Reply:
Valid Nit! Will fix